boa_cli/src/main.rs

```diff
@@ -2,29 +2,30 @@
 #![warn(clippy::perf)]
 #![allow(clippy::cognitive_complexity)]
 
+use boa::builtins::console::log;
 use boa::{exec::Executor, forward_val, realm::Realm};
+use std::io;
 use std::{fs::read_to_string, path::PathBuf};
 use structopt::StructOpt;
-
 /// CLI configuration for Boa.
 #[derive(Debug, StructOpt)]
 #[structopt(author, about)]
 struct Opt {
     /// The javascript file to be evaluated.
-    #[structopt(name = "FILE", parse(from_os_str), default_value = "tests/js/test.js")]
+    #[structopt(name = "FILE", parse(from_os_str))]
```

Comment:
So if i understand it by default boa_cli will now be a shell, and we need to pass the file if we want to keep the same functionality as before?
Im personally OK with that as it matches up with Node etc

Reply:
Yup! It should be functionally equivalent. I just tested running node locally to check and passing no arguments (just calling `node`) opens an interactive shell.